packages/openapi-ts/src/index.ts

```diff
@@ -184,6 +188,43 @@ const getTypes = (userConfig: UserConfig): Config['types'] => {
   return types;
 };
 
+const getInstalledDependencies = (): Dependencies => {
+  const toReducedDependencies = (p: PackageDependencies): Dependencies =>
+    [p.dependencies, p.devDependencies].reduce(
```

Comment:
Will this work if the devDependencies key doesn't exist?

Reply:
Just updated to make both dependencies/devDependencies optional